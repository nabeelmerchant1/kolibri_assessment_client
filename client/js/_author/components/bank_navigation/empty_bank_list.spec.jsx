import React            from 'react';
import { shallow }      from 'enzyme';
import EmptyBankList    from './empty_bank_list';

describe('empty_bank_list component', () => {
  let props;
  let result;

  beforeEach(() => {
    props = {};
    result = shallow(<EmptyBankList {...props} />);
  });

  it('renders the component as a snapshot', () => {
    expect(result).toMatchSnapshot();
  });
});
