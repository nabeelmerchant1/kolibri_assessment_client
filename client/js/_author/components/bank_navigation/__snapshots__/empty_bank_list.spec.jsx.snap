// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`empty_bank_list component renders the component as a snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <table
    className="au-c-table is-empty"
>
    <tbody>
        <tr>
            <td>
                <div>
                    Add a new assessment to get started.
                </div>
            </td>
        </tr>
    </tbody>
</table>,
  "nodes": Array [
    <table
      className="au-c-table is-empty"
>
      <tbody>
            <tr>
                  <td>
                        <div>
                              Add a new assessment to get started.
                        </div>
                  </td>
            </tr>
      </tbody>
</table>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <EmptyBankList />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <table
          className="au-c-table is-empty"
>
          <tbody>
                    <tr>
                              <td>
                                        <div>
                                                  Add a new assessment to get started.
                                        </div>
                              </td>
                    </tr>
          </tbody>
</table>,
        "_debugID": 2,
        "_renderedOutput": <table
          className="au-c-table is-empty"
>
          <tbody>
                    <tr>
                              <td>
                                        <div>
                                                  Add a new assessment to get started.
                                        </div>
                              </td>
                    </tr>
          </tbody>
</table>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <EmptyBankList />,
}
`;
